HELP.md

```diff
@@ -0,0 +1,124 @@
+# Overview
+
+- [Installation](#installation)
+  - [cnquery](#cnquery)
+  - [cnspec](#cnspec)
+- [Package Information](#package-information)
+- [Kubernetes](#kubernetes)
+
+# Installation
+
+## cnquery
+
+###  via Shell Script (Linux and macOS)
+
+[`https://install.mondoo.com/sh/cnquery`](https://install.mondoo.com/sh/cnquery)
+
+```bash
+bash -c "$(curl -sSL https://install.mondoo.com/sh/cnquery)"
+```
+
+To login with Mondoo Platform:
+
+```bash
+cnquery login -t 'eyJh...llZ4BW'
+```
+
+### via Powershell (Windows)
+
+[`https://install.mondoo.com/ps1/cnquery`](https://install.mondoo.com/ps1/cnquery)
+
+```powershell
+Set-ExecutionPolicy Unrestricted -Scope Process -Force;
+[System.Net.ServicePointManager]::SecurityProtocol = [System.Net.ServicePointManager]::SecurityProtocol -bor 3072;
+iex ((New-Object System.Net.WebClient).DownloadString('https://install.mondoo.com/ps1/cnquery'));
+Install-Mondoo -Product cnquery;
+```
+
+## cnspec
+
+###  via Shell Script (Linux and macOS)
+
+[`https://install.mondoo.com/sh/cnspec`](https://install.mondoo.com/sh/cnspec)
+
+```bash
+bash -c "$(curl -sSL https://install.mondoo.com/sh/cnspec)"
+```
+
+To login with Mondoo Platform:
+
+```bash
+cnspec login -t 'eyJh...llZ4BW'
+```
+
+### via Powershell (Windows)
+
+[`https://install.mondoo.com/ps1/cnspec`](https://install.mondoo.com/ps1/cnspec)
+
+```powershell
+Set-ExecutionPolicy Unrestricted -Scope Process -Force;
+[System.Net.ServicePointManager]::SecurityProtocol = [System.Net.ServicePointManager]::SecurityProtocol -bor 3072;
+iex ((New-Object System.Net.WebClient).DownloadString('https://install.mondoo.com/ps1/cnspec'));
+Install-Mondoo -Product cnspec;
+```
```

Comment:
Shoud we add the login part from L51 also here?

Reply:
I think that is a great idea.